battletracker/scripts/unitsummary.lua

```diff
@@ -14,21 +14,24 @@ function onInit()
 	DB.addHandler(DB.getPath(nodeUnit, "ancestry"), "onUpdate", updateSummary);
 	DB.addHandler(DB.getPath(nodeUnit, "type"), "onUpdate", updateSummary);
 
-	nodeUnit = getDatabaseNode(); -- Color is only tracked in the CT node
-	onColorChanged(DB.getChild(nodeUnit, "color"));
-	DB.addHandler(DB.getPath(nodeUnit, "color"), "onUpdate", onColorChanged);
+	-- Color and ID are only tracked in the CT node
+	onColorChanged(DB.getChild(getDatabaseNode(), "color"));
```

Comment:
Why change to multiple getDatabaseNode() calls? For context, MoonWizard has stated that getting a node is an allocating operation.